src/Microsoft.Identity.Client/Platforms/Android/EmbeddedWebview/AuthenticationAgentActivity.cs

```diff
@@ -113,7 +113,6 @@ public override void OnLoadResource(WebView view, string url)
 
             }
 
-            [Obsolete]
             public override bool ShouldOverrideUrlLoading(WebView view, string url)
```

Comment:
weren't you going to move this to internal?  it's still marked as public.

Reply:
The class is private, I marked it obsolete by mistake.